go/oasis-test-runner/oasis/network.go

```diff
@@ -638,8 +644,16 @@ func (net *Network) startOasisNode(
 
 	cfg.Common.DataDir = node.DataDir()
 	cfg.Common.Log.Level = make(map[string]string)
```

Comment:
Is this necessary? It looks like `config.DefaultConfig` already initializes it. But maybe `Marshal` sets it back to `nil` :shrug:

Reply:
Yeah, this is just to make sure it's really initialized (it might not be when loading from file if the fields aren't set).